plugins/net_plugin/net_plugin.cpp

```diff
@@ -3150,10 +3150,8 @@ namespace eosio {
    }
 
    void net_plugin_impl::plugin_shutdown() {
-         in_shutdown = true;
-
-         connections.close_all();
-         thread_pool.stop();
+      in_shutdown = true;
```

Comment:
What is the purpose of `in_shutdown`? From a quick scan, looks like it could never be `true` anywhere we check for it.

Reply:
okay I see how it can be `true`, but is this necessary?

Reply:
No, I don't think it is. I'll remove it.